requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/sourcegraph/pydep.git@3230f0b3b71b4b779888c9e6c4aa6a54c5749d55
+git+https://github.com/sourcegraph/pydep.git
```

Comment:
Can we keep the commit hash to avoid silently breaking things due to dependency changes?